.gitlab-ci.yml

```diff
@@ -196,6 +196,23 @@ package-arm:
   variables:
     ARCH: arm64
 
+package-oci:
+  stage: package
+  extends: .package-oci
+  rules:
+    - if: $DOTNET_PACKAGE_VERSION
+      when: on_success
+    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/'
+      when: manual
+      allow_failure: false
```

Comment:
Is this safe to start pushing them immediately (on next release)? Our current release process will run this automatically

Reply:
Yes, it's safe. The versions can only be used by the updater after we add them to the remote-config catalog, so releasing doesn't have an immediate effect